purchase_request/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Purchase Request",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
-    "version": "16.0.1.0.3",
+    "version": "16.0.1.0.4",
```

Comment:
FYI, such change is useless, as it will be done automatically by the bot merging the PR.

Reply:
Thanks! Change done!